assignments/array-methods.js

```diff
@@ -54,30 +57,48 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 {"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
 
 // ==== Challenge 1: Use .forEach() ====
-// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
+// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName.
 let fullName = [];
+runners.forEach((runner)=> fullName.push(`${runner.first_name} ${runner.last_name}`));
```

Comment:
With Arrow Function Syntax, if you only have ONE parameter, you don't need to wrap them in parenthesis! they are optional! (two or more params would require parenthesis though!)

runners.forEach( runner => fullName.push(`${runner.first_name} ${runner.last_name}`));